src/main/java/com/hanghae99/wanted/service/ApiTagService.java

```diff
@@ -0,0 +1,31 @@
+package com.hanghae99.wanted.service;
+
+import com.hanghae99.wanted.web.dto.response.TagResponse;
+import com.hanghae99.wanted.web.entity.tag.TagRepository;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @Created by Bloo
+ * @Date: 2021/07/19
+ */
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class ApiTagService {
+
+    private final TagRepository tagRepository;
+
+    @Transactional (readOnly = true)
+    public List<TagResponse> findTagByCategoryId ( Long id ) {
+        return tagRepository.findAllByCategoryId(id)
```

Comment:
-Passing Thought-
NullPointerException could be thrown in case of another kind of service.
But ours is not the case as Wanted-Server-Application tags are injected in the first place.